server/s3server.cc

```diff
@@ -1258,6 +1259,9 @@ int main(int argc, char **argv) {
           g_option_instance->get_bucket_metadata_cache_expire_sec(),
           g_option_instance->get_bucket_metadata_cache_refresh_sec()));
 
+  S3DataUsageCache::get_instance()->set_max_cache_size(
+    g_option_instance->get_bucket_metadata_cache_max_size());
```

Comment:
Let's have separate option -- since this is not really a bucket-related cache any more.

New parameter name: `S3_DATA_USAGE_ACCOUNTS_CACHE_MAX_SIZE`.

Reply:
Done.